doc/api/errors.md

```diff
@@ -597,6 +597,7 @@ Used when `Console` is instantiated without `stdout` stream or when `stdout` or
 Used when the native call from `process.cpuUsage` cannot be processed properly.
 
 <a id="ERR_DNS_SET_SERVERS_FAILED"></a>
+### ERR_DNS_SET_SERVERS_FAILED
```

Comment:
It seems that I missing some titles for `errors.md` in #14212 . So I added the missing titles in this PR